### PR TITLE
webui: check existence of payload image when running testing VM

### PR DESCRIPTION
**Pull request by rvykydal**

### CODE REVIEW

File: ui/webui/test/machine_install.py [hunk 1]
```diff
@@ -108,7 +108,11 @@ def start(self):
 
         self.http_updates_img_port = self._serve_updates_img()
 
-        payload_cached_name, http_payload_port = self._serve_payload()
+        payload_path = os.path.join(BOTS_DIR, "./images/fedora-rawhide-anaconda-payload")
+        if not os.path.exists(payload_path):
+            raise FileNotFoundError(f"Missing payload file {payload_path}. (Maybe try running 'make payload').")
```
Comment (KKoukiou) [inline ui/webui/test/machine_install.py:113]: Perfect - just I would be more assertive on the suggestion. 
  raise FileNotFoundError(f"Missing payload file {payload_path}; use 'make payload').")

Not strong opinion though.

**Review by KKoukiou — APPROVED** [review]

File: ui/webui/test/machine_install.py (hunk 1 above)
Comment (rvykydal) [reply ui/webui/test/machine_install.py]: updated

**Review by KKoukiou — APPROVED** [review]

**Review by M4rtinK — APPROVED** [review]
Looks good to me as well - nice improvement! :)

**Pull request merged**